extensions/amp-story/1.0/amp-story-system-layer.css

```diff
@@ -307,9 +256,8 @@
   display: block !important;
 }
 
-.i-amphtml-story-system-layer-buttons button[disabled] {
-  opacity: 0.3 !important;
-  cursor: initial !important;
+.i-amphtml-story-system-layer-buttons button[disabled][disabled] {
```

Comment:
I don't think i've seen `[disabled][disabled]` before.
Out of curiosity, how is this different than `[disabled]`?